public/js/main.js

```diff
@@ -83,6 +83,8 @@ document.getElementsByClassName("form-input refine")[0].addEventListener("click"
     document.getElementsByClassName("box refine")[0].style.display = document.getElementsByClassName("box refine")[0].style.display == "none" ? "block" : "none";
     if(document.getElementsByClassName("form-input refine-searchbox")[0].value != document.getElementsByClassName("form-input search-box")[0].value)
         document.getElementsByClassName("form-input refine-searchbox")[0].value = document.getElementsByClassName("form-input search-box")[0].value;
-    e.preventDefault();
+    if(document.getElementsByClassName("form-input refine-category")[0].value != document.getElementsByClassName("form-input form-category")[0].value)
+        document.getElementsByClassName("form-input refine-category")[0].value = document.getElementsByClassName("form-input form-category")[0].value;
```

Comment:
This won't work for a select, you have to range over each option tag and apply the selected attribute on the right one